reference/api/tasks.md

```diff
@@ -117,17 +117,23 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `taskDeletion`
 
+| Name                  | Description                                                                                                                                                                                         |
+| :-------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`matchedTasks`**    | The number of tasks that can be deleted based on the request. If the API key doesn’t have access to any of the indexes specified in the request, those tasks will not be included in `matchedTasks` |
```

Comment:
I find this sentence a little bit misleading:
> The number of tasks that can be deleted based on the request.

Because if you, for example, ask to delete all the enqueued tasks (`?statuses=enqueued`), none of these tasks can be deleted, but they're still matched.

Maybe something in the form of « The number of tasks that matched your filter. » would be more clear?